private4/index.php

```diff
@@ -419,13 +412,13 @@ function lastOne() {
       location.hash = "side" + (data.page + 1);
   if (!location.hash.replace(/^#/, '')) {
   location.hash = "side" + (viewer.currentPage() + 1);
-}    
+}
     });
 
-<!-- Aktiverer separat url (hash) -->
-    viewer.addHandler('open', function() {
-  updateForHash();
-});
+    <!-- Aktiverer separat url (hash) -->
+    viewer.addOnceHandler('open', function() {
+      updateForHash();
```

Comment:
I fixed up the indenting of these few lines... proper indenting helps make the code easier to read.